plugins/coverbrowser.koplugin/covermenu.lua

```diff
@@ -139,17 +139,17 @@ function CoverMenu:updateItems(select_number)
                 item:update()
                 if item.bookinfo_found then
                     logger.dbg("  found", item.text)
+                    self.show_parent.dithered = item._has_cover_image
                     local refreshfunc = function()
                         if item.refresh_dimen then
                             -- MosaicMenuItem may exceed its own dimen in its paintTo
                             -- with its "description" hint
-                            return "ui", item.refresh_dimen, item._has_cover_image
+                            return "ui", item.refresh_dimen, self.show_parent.dithered
                         else
-                            return "ui", item[1].dimen, item._has_cover_image
+                            return "ui", item[1].dimen, self.show_parent.dithered
```

Comment:
Still not deep into your UIManager changes, so just asking: is that allright to set `self.show_parent.dithered` (which will stick) when you're doing a refresh of only a single thumbnail?
(Or is it allright just because after that, the full 9x9 CoverMenu/FM has a least one image, so it can stay dithered from now on?)